doc/user_guide/user_guide.md

```diff
@@ -62,6 +62,14 @@ This is why **you should not use test containers in a production environment**.
 
 The Exasol test container provides access to buckets in BucketFS. This is useful if your tests need to work with files in buckets. If you for example want to test a UDF script, you can upload it prior to the test using a `Bucket` control object. 
 
+### Understanding Bucket Contents
+
+One thing you need to know about buckets objects inside a bucket are not stored in a hierarchical directory structure. Instead they are flat files that can have slashes in their names so that it looks like a directory structure at first glance.
```

Comment:
Suggestion:

`... objects inside a bucket is that they are not stored in a hierarchical directory structure.`

Reply:
Fixed.